filebeat/docs/filebeat-general-options.asciidoc

```diff
@@ -50,6 +50,25 @@ Examples:
 filebeat.registry_file_permissions: 0600
 -------------------------------------------------------------------------------------
 
+[float]
+==== `registry_flush`
+
+
+The registry flush timeout. The registry is written to disk if the oldest
```

Comment:
Please consider this for first sentence:
"The timeout value that controls when registry entries are written to disk
(flushed)."